### DISCUSSION TITLE
CopyInterpreterV2.list_files can be same with InterpreterCommon.list_files?

### DISCUSSION THREAD

**Discussion by bohutang** [databendlabs/databend#7754] [General]
https://github.com/datafuselabs/databend/blob/d7a43c9d56ec1acc9751768146b4c3fff9770459/src/query/service/src/interpreters/interpreter_copy_v2.rs#L158-L211

https://github.com/datafuselabs/databend/blob/d7a43c9d56ec1acc9751768146b4c3fff9770459/src/query/service/src/interpreters/interpreter_common.rs#L182-L274

**Comment by bohutang** [comment]
cc @Xuanwo  for suggestions to improve it

**Reply by Xuanwo** [reply]
Yes, we can reuse some of the code. Let's me do this refactor later.

**Reply by Xuanwo** [reply]
Tracked at https://github.com/datafuselabs/databend/issues/7755
